src/Components/Directory.Build.props

```diff
@@ -9,6 +9,10 @@
     <OutputPath />
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnableTypeScriptNuGetTarget>true</EnableTypeScriptNuGetTarget>
```

Comment:
We should also have a comment explaining what this is needed for. I searched the web, and this property doesn't appear to be publicly documented, so it's unclear what the consequences of this are or whether it should be more closely scoped to something being changed in this PR.

Reply:
https://github.com/dotnet/aspnetcore/issues/18980